tasks/system_probe.py

```diff
@@ -1598,8 +1599,12 @@ def save_test_dockers(ctx, output_dir, arch, windows=is_windows):
     if windows:
         return
 
+    # only download images not present in preprepared vm disk
+    resp = requests.get('https://dd-agent-omnibus.s3.amazonaws.com/kernel-version-testing/rootfs/docker.ls')
+    docker_ls = set([line for line in resp.text.split('\n') if line.strip()])
+
     images = _test_docker_image_list()
-    for image in images:
+    for image in (images-docker_ls):
```

Comment:
nitpick: I'd add some spacing to make it clear that it `images` - `docker_ls` and not a single identifier